src/ServicePulse.Host/vue/src/components/failedmessages/MessageGroupList.vue

```diff
@@ -166,7 +166,7 @@ async function saveDeleteGroup(group) {
   // We've started a delete, so increase the polling frequency
   changeRefreshInterval(1000);
 
-  saveDeleteNote(group, true); // delete comment note when group is archived
+  await saveDeleteNote(group, true); // delete comment note when group is archived
```

Comment:
this is changing behaviour to wait for the delete note promise to complete before continuing. If this isn't necessary then it will affect performance